app/assets/stylesheets/common/components/topic-filter-education.scss

```diff
@@ -0,0 +1,27 @@
+@use "lib/viewport";
+
+.empty-topic-filter-education {
+  margin: 0 auto;
+  width: 75%;
+  text-align: center;
+
+  &__preferences-hint {
+    margin-top: 2rem;
+  }
+
+  &__image {
+    margin-top: 6rem;
+
+    svg {
+      width: 200px;
```

Comment:
Should we maybe have an adjustment for mobile? Using vw units? I think slightly scaled up is nicer on mobile.